src/pages/docs/step-ca/provisioners.mdx

```diff
@@ -150,43 +150,55 @@ These commands require you to sign in as an Admin user.
 
 ### Enable Remote Provisioner Management
 
-To enable remote provisioner management, update your `"authority"` block in `ca.json` to include the following:
+#### On a New CA
 
-```json
-"authority": {
-		"enableAdmin": true,
-		"provisioners": []
-},
-```
+To enable remote provisioner management on a new CA, pass `--remote-management` to `step ca init`. This will:
 
-The `"provisioners"` list must be empty, and `"enableAdmin"` must be set to `true`.
+* Add an initial [JWK](#jwk) provisioner called Admin JWK to the database.
+* Create an initial Super Admin user, with default username `step`, and link it to the Admin JWK provisioner.
 
-### Bootstrap Remote Provisioner Management
+#### On an Existing CA
 
-During this step:
-* An initial [JWK](#jwk) provisioner called Admin JWK will be added to the database.
-* An initial Super Admin user, with username `step`, will be created and linked to the Admin JWK provisioner.
-
-"Admins" can remotely manage provisioners.
-"Super Admins" are Admins that can also manage the list of Admins for the CA.
+To migrate your `ca.json` provisioners to the database, and to enable remote provisioner management on an existing CA:
 
 1. Stop `step-ca` if it is running.
-2. Start `step-ca` from the command line.
+2. Update your `"authority"` block in `ca.json` to include the following:
+
+  ```json
+  "authority": {
+  		"enableAdmin": true,
+          "provisioners": [
+            ...
+          ]
+  },
+  ```
+ 
+3. Start `step-ca` from the command line.
+
+   When `step-ca` starts up, it will:
+   
+   * Migrate the provisioners from your `ca.json` to the database
+   * Add an initial [JWK](#jwk) provisioner called Admin JWK to the database.
```

Comment:
The current behavior is that the first JWK provisioner found in `ca.json` will be used for admin purposes. Usually this will be the default JWK provisioner. This also means that the existing encrypted key can be used to authenticate, also if it's no longer stored in the CA. If there's no JWK provisioner, a new one will be created and that will be used for admin purposes. It may be a bit surprising, I guess, but there's log output that explains what happens.